src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.scss

```diff
@@ -54,6 +54,19 @@
   &.split-screen {
     background: linear-gradient(to bottom,  #00a99c 0%,#00a99c 50%,#265753 50%,#265753 100%);
   }
+
+  &.replay {
+    background-color: #8d1010;
+  }
+
+  &.video-clip-voice-over {
+    background: linear-gradient(to bottom,  #1769ff 0%,#1769ff 50%, #005919 50%,#005919 100%);
+  }
+
+  &.replay-voice-over {
+    background: linear-gradient(to bottom,  #8d1010 0%,#8d1010 50%, #005919 50%,#005919 100%);
+  }
```

Comment:
Instead of using two classes here, we could use two classes instead. That would also make it possible to make a default voice over styling, e.g.:
```scss
  &.voice-over {
    /* default voice over styling */
  }

  &.replay {
    /* default replay styling */

    &.voice-over {
      /* replay voice over styling */
    }
  }
  
    &.server {
    /* default server styling */

    &.voice-over {
      /* replay voice over styling */
    }
  }
```